console/src/components/Services/Settings/Actions.js

```diff
@@ -80,6 +82,20 @@ export const exportMetadata = (successCb, errorCb) => (dispatch, getState) => {
     });
 };
 
+export const exportSchema = () => (dispatch, getState) => {
```

Comment:
```suggestion
export const exportSchema => (dispatch, getState) => {
```

It's not taking any arguments anyway.